### DISCUSSION TITLE
Hostpool is not reflecting in the AVD web client

### DISCUSSION THREAD

**Discussion by ghost** [Azure/avdaccelerator#684] [General]
Hello 

We have three security groups in Entra ID, security group 1 shows users inside them and its hostpool successfully reflects on AVD web client. The security group 2 is inheriting users from security group 3. But security group 2 and 3 doesn’t have any users in Entra ID. Do we have to manually added users into security group or nested group across security groups is allowed in AVD?

Can someone please help?

**Comment by danycontre** [comment]
@Wasim49  apologies didn't get notifications for this discussion.

Is this still valid?

**Comment by ghost** [comment]
Hi @danycontre - the issue has been solved, thank you for commenting on the thread
